dynamicforms/vue/src/mixins/translationsMixin.js

```diff
@@ -0,0 +1,9 @@
+const translationsMixin = {
+  methods: {
+    gettext(str) {
+      return str;
```

Comment:
tole bi moralo preverit, če je window.gettext defined in potem vrnit njega. Če pa ni, pa str.

Reply:
popravljeno